docs/api/browser-window.md

```diff
@@ -1162,6 +1162,9 @@ Returns `boolean` - Whether the window's document has been edited.
 * `opts` Object (optional)
   * `stayHidden` boolean (optional) -  Keep the page hidden instead of visible. Default is `false`.
   * `stayAwake` boolean (optional) -  Keep the system awake instead of allowing it to sleep. Default is `false`.
+  * `outputSize` Object (optional)
```

Comment:
We should specify the units being used for both width and height.

Reply:
added: _The desired width and height in pixels (positive numbers) of the returned image_